db-cpdb/bash/01_dataloading.sh

```diff
@@ -2,39 +2,40 @@
 source bash/config.sh
 
 # Reference tables
-psql "${BUILD_ENGINE}" -f "sql/_create.sql"
+run_sql_file "sql/_create.sql"
+create_source_data_table
 
 # Spatial boundaries
-import dcp_boroboundaries_wi &
-import dcp_stateassemblydistricts &
-import dcp_ct2020 &
-import dcp_congressionaldistricts &
-import dcp_cdboundaries &
-import dcp_statesenatedistricts &
-import dcp_municipalcourtdistricts &
-import dcp_school_districts &
-import dcp_trafficanalysiszones &
-import dcp_councildistricts &
-import nypd_policeprecincts &
-import fdny_firecompanies &
+import_recipe dcp_boroboundaries_wi &
+import_recipe dcp_stateassemblydistricts &
+import_recipe dcp_ct2020 &
+import_recipe dcp_congressionaldistricts &
+import_recipe dcp_cdboundaries &
+import_recipe dcp_statesenatedistricts &
+import_recipe dcp_municipalcourtdistricts &
+import_recipe dcp_school_districts &
+import_recipe dcp_trafficanalysiszones &
+import_recipe dcp_councildistricts &
+import_recipe nypd_policeprecincts &
+import_recipe fdny_firecompanies &
 
 # Building and lot-level info
-import dcp_mappluto_wi &
-import dcp_facilities 20210811 &
-import doitt_buildingfootprints &
+import_recipe dcp_mappluto_wi &
+import_recipe dcp_facilities 20210811 & # why pinned to this version?
```

Comment:
@AmandaDoyle @damonmcc any thoughts on why facdb is pinned to this version in cpdb build?

Reply:
hmm I'm not sure, worth checking with Amanda

my only idea is that it may be joined onto some other source data that isn't updated very frequently, so an older FacDB version is more useful than the latest?

Reply:
Looks like we just pinned FacDB to the latest published version.  We do not need to pin to a specific version of FacDB 
https://github.com/NYCPlanning/db-cpdb/commit/3ff76c0f600365cc20a1493a4decf24e5b436bfa

Reply:
Gotcha. I'm just goin to unpin then, default to latest if that sounds good